server/index.js

```diff
@@ -34,6 +34,15 @@ app.get('/*', (req, res) => {
   res.sendFile(path.resolve(__dirname, '../public/index.html'));
 });
 
+// server connection
+const PORT = process.env.PORT || 3001;
+
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
Nice use of environment variables; I appreciated this conversation on Slack / learning that NODE_ENV is set to 'test' during testing

Reply:
> All in all your tests look solid. You are testing all the success scenarios, but I might throw in an error scenario or two. Also, I am wondering where the file which contains the implementation of your CRUD routes is.

Yup, that makes total sense! I'll pull this request into main, but I will definitely work on error scenarios and add them in.